### DISCUSSION TITLE
Page generation should include a CSP

### DISCUSSION THREAD

**Discussion by scottcwilson** [zencart/zencart#6119] [Ideas]
As some point, I suspect browsers will make this mandatory. 

https://developer.chrome.com/docs/lighthouse/best-practices/csp-xss/

**Comment by lat9** [comment]
A page _should_ contain a Content Security Policy, but having attempted one on a fairly modified Zen Cart, I'll say that it's not as straight-forward as that article would make one believe!

**Comment by scottcwilson** [comment]
Oh I agree.  It's a great way to break your site if it's not done correctly.  

We'll have to noodle on it to see if we can come up with something reasonable.

**Comment by scottcwilson** [comment]
Migrating to discussion.
